src/bridge/cockpitdbussetup.c

```diff
@@ -218,7 +218,7 @@ setup_prepare_passwd1 (GVariant *parameters,
   if (!g_str_equal (mechanism, "passwd1"))
     {
       g_dbus_method_invocation_return_error (invocation, G_DBUS_ERROR, G_DBUS_ERROR_NOT_SUPPORTED,
-                                             _("Unsupported setup mechanism: %s"), mechanism);
+                                             N_("Unsupported setup mechanism: %s"), mechanism);
```

Comment:
I don't think this works for these, because mechanism isn't known in advance the front end won't have a translation for it. Maybe that's ok though.